src/components/application_manager/include/application_manager/commands/hmi/response_from_hmi.h

```diff
@@ -66,8 +64,9 @@ class ResponseFromHMI : public CommandImpl {
    * @param function_id HMI request ID
    * @param msg_params HMI request msg params
    */
-  void CreateHMIRequest(const hmi_apis::FunctionID::eType& function_id,
-                        const smart_objects::SmartObject& msg_params) const;
+  void CreateHMIRequest(
+      const hmi_apis::FunctionID::eType& function_id,
+      const NsSmartDeviceLink::NsSmartObjects::SmartObject& msg_params) const;
```

Comment:
@wolfylambova22, why do you replace `smart_objects` namespace alias on a full namespace ` name NsSmartDeviceLink::NsSmartObjects`?